File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/SupplierInfoDto.java
package com.sudo248.orderservice.controller.order.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierInfoDto implements Serializable {
    private String supplierId;
    private int ghnShopId;
    private String name;
    private String avatar;
    private String contactUrl;
    private AddressDto address;
    private float rate;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/discovery/SupplierInfoDto.kt
package com.sudo248.sudoo.data.dto.discovery

import com.sudo248.sudoo.data.dto.user.AddressDto

data class SupplierInfoDto(
    val supplierId: String,
    val name: String,
    val avatar: String,
    val contactUrl: String,
    val rate: Float,
    val address: AddressDto
)
